changelogs/fragments/349-skip-crds.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - helm - add a ``skip_crds`` option to skip the installation of CRDs when installing or upgrading a chart (https://github.com/ansible-collections/community.kubernetes/issues/296).
```

Comment:
Could you please add a test for this change? Thanks. Rest LGTM

Reply:
Do you have some thoughts on how I might do that? Do you know of a chart that has CRDs?

Reply:
You can use this [simple helm chart](https://github.com/Azure-Samples/helm-charts/tree/master/chart-source/tweet-factory-operator) or create your own based upon this.